instance/templates/instance/ansible/vars.yml

```diff
@@ -30,6 +30,7 @@ EDXAPP_BULK_EMAIL_DEFAULT_FROM_EMAIL: '{{ instance.email }}'
 
 # Misc
 EDXAPP_TIME_ZONE: 'UTC'
+edxapp_theme_name: '{{ instance.theme_name }}'
```

Comment:
That means a theme will always be activated. What if we don't want any theme applied?